.shippable.yml

```diff
@@ -33,11 +33,11 @@ build:
     - _make CFG_WITH_PAGER=y
     - _make CFG_WITH_PAGER=y CFG_WITH_LPAE=y
     - _make CFG_WITH_LPAE=y
-    - _make CFG_WITH_STATS=y
     - _make CFG_RPMB_FS=y
     - _make CFG_RPMB_FS=y CFG_RPMB_TESTKEY=y
     - _make CFG_REE_FS=n CFG_RPMB_FS=y
-    - _make CFG_WITH_USER_TA=n CFG_CRYPTO=n CFG_SE_API=n CFG_PCSC_PASSTHRU_READER_DRV=n
+    - _make CFG_WITH_USER_TA=n CFG_CRYPTO=n CFG_SE_API=n
```

Comment:
As these targets aren't executed here, why do we need to build `CFG_PCSC_PASSTHRU_READER_DRV=y` separatebly?

Reply:
No particular reason. Updated to simply replace `=n` with `=y`.